### PR TITLE
do not log the body of a request in auditLogs per default

### PR DESCRIPTION
**Pull request by szuecs**
Signed-off-by: Sandor Szücs <sandor.szuecs@zalando.de>

### CODE REVIEW

File: cluster/manifests/skipper/daemonset.yaml [hunk 1]
```diff
@@ -59,6 +59,7 @@ spec:
           - "-lb-healthcheck-interval=3s"
           - "-metrics-flavour=codahale,prometheus"
           - "-enable-connection-metrics"
+          - "--max-audit-body=0"
```
Comment (aryszka) [inline cluster/manifests/skipper/daemonset.yaml]: should be a single dash ("-") instead of double ("--")

Comment (szuecs) [reply cluster/manifests/skipper/daemonset.yaml]: thanks fixed

**Comment by aryszka** [comment]
:+1:

**Comment by szuecs** [comment]
:+1:

**Pull request merged**